SETUP_ORACLE.md

```diff
@@ -68,7 +68,27 @@ You need to have Docker installed and running. You also need to be able to [run
 
 1. From this repository, do `make oracle-database` and wait to see *DATABASE IS READY TO USE!*.
 
-2. Finally initialize the database with some seed data by running: `DATABASE_URL="oracle-enhanced://rails:railspass@127.0.0.1:1521/systempdb" ORACLE_SYSTEM_PASSWORD=threescalepass NLS_LANG=AMERICAN_AMERICA.UTF8 USER_PASSWORD=123456 MASTER_PASSWORD=123456 MASTER_ACCESS_TOKEN=token bundle exec rake db:drop db:create db:setup`
+    2. This will create a new user to stablish regular connections with the Oracle Database. For that, Oracle's SYSTEM user must be used.
+        Alternatively, the `ORACLE_SYSTEM_PASSWORD` ENV variable might be omitted and, in this case, a valid user must be provided.
+        To create such user and GRANT it the necessary permissions, you might do the following:
```

Comment:
Wording not clear here. `ORACLE_SYSTEM_PASSWORD` is missing from point 3. We can't omit `ORACLE_SYSTEM_PASSWORD` if it was not present. What I assume is that points 2 and 3 have to be switched and the `Alternatively` word must go more to the front. Or something else if you meant so.

Reply:
Updated in [11a06bb](https://github.com/3scale/porta/pull/3195/commits/11a06bb241f66503d2d9dcd47e2becd4caa5b97a).

Reply:
ok, maybe then just split `Alternatively ...` into a separate sub-bullet point so that Point 1 will have two bullets. I'm asking because it was confusing to me to understand what `this` was.

Sorry for nitpicking

Reply:
Updated in [b20b02c](https://github.com/3scale/porta/pull/3195/commits/b20b02caec0a231c4a77cbf773f9c4178855b5e9).